features/step_definitions/comment_actions/create_comment.rb

```diff
@@ -0,0 +1,27 @@
+Given(/^Post page/) do
+  user = create(:user, name: 'Admin', email: 'admin@example.com', password: 'password')
```

Comment:
It looks like FactoryGirl is used here.It will not work with kraft ORM as I know. Isn't it better to avoid usage of ActiveRecord dependent things if we are going to migrate to SimolyRecord ?